plugins/repository-gcs/src/main/java/org/elasticsearch/repositories/gcs/GoogleCloudStorageBlobStore.java

```diff
@@ -427,7 +445,9 @@ public void success(Boolean result) {
                             @Override
                             public void error(StorageException exception) {
                                 if (exception.getCode() != HTTP_NOT_FOUND) {
-                                    failedBlobs.add(blob);
+                                    if (failedBlobs.size() < 10) {
```

Comment:
nit: extract `10` to a named constant?

Reply:
I added a comment instead, seemed like it would achieve the same with less code? :)